sdks/typescript/client-typescript-eventsource/README.md

```diff
@@ -34,30 +34,59 @@ It should look similar to this: ```https://vrtfs.demo.featurehub.io/features/def
 
 ![Service account](images/service-account-copy.png) 
 
-## Mechanisms for use
+## The FeatureHub repository
 
-Unlike the Java SDK, there are four ways to use this library due to the more _user_ based interaction that your 
-application will operate under.
+### Overview
 
-### 1. Real-time updates, recommended for servers
+The FeatureHub repository is a single class that holds and tracks features in your system. It gets features delivered
+to it to process, tracks changes, and allows you to find and act on features in a useful way. 
+It also sends events out in certain circumstances.
 
-In this mode, you will make a connection to the FeatureHub Edge server, and any updates to any events will come
-through to you, updating the feature values in the repository. We recommend this technique for servers, 
-but for Web it may not be the best choice
-as you may not want your users to see updates in the browser immediately but rather serve it on-demand. 
+### Creating the FeatureHub repository
 
-A typical strategy will be:
+Although one exists and is exported for your use (`featureHubRepository`), you can create others if you wish. The class is simply 
```

Comment:
do you mean you can create multiple repositories? I am not sure I get why would someone need that?

Reply:
listening to multiple applications.